backend/routes/auth/instagram.js

```diff
@@ -0,0 +1,14 @@
+const router = require("express").Router();
+const dotenv = require("dotenv").config();
```

Comment:
I think we only need to load the config once (though it doesn't hurt to load try and load it again). It's already being loaded currently in app.js, haha but should be transferred to server.js, because dotenv needs to be loaded 1 time before it can be used.

Reply:
Thanks. I thought we had to write it in each subrouter. I will review this